src/components/index.tsx

```diff
@@ -1,11 +1,17 @@
-
 /*
  *
  * @section HeroSection
  *
  */
 export { HeroSection } from "./sections/Hero.section";
 
+/*
+ *
+ * @section FAQSection
```

Comment:
```suggestion
 * @section InfoSection
```

Reply:
Nice catch! I must have forgotten to update that. Fixed it